skupstina_django/scraper/scrape_utils.py

```diff
@@ -0,0 +1,127 @@
+import re
+import requests
+from bs4 import BeautifulSoup
+from skupstina.models import Category, Source, Item, Act
+
+root_url = 'http://web.zagreb.hr'
+
+
+def parse_subjects_list(url):
+    site = requests.get(url).content
+    soup = BeautifulSoup(site, 'html.parser')
+    els = soup.select('.centralTD ul ol li')
+    print(len(els), ' elements')
+    subjects = []
+    for el in els:
```

Comment:
generally when you're coding it's good to try and give semantic variable names. It's easier for you and others when they read this in the future. I'd write e.g. (not sure if these are the best names, because I don't know which part of the page these elements are on, but I'd try to describe whatever they are on the page):

```
table_items = soup.select('.centralTD ul ol li')
for item in table_items:
    ....
```

Reply:
I didn't change the variable names from the previous code. But you're right, I will update them to made them more readable.